examples/train.py

```diff
@@ -374,6 +382,37 @@ def trial_str_creator(trial):
         exp_dict['upload_dir'] = s3_string
     tune.run(**exp_dict, queue_trials=False, raise_on_failed_trial=False)
 
+    if flags.upload_graphs:
+        print('Generating experiment graphs and uploading them to leaderboard')
+        submitter_name, strategy_name = flags.upload_graphs
+
+        # grab checkpoint path
+        for (dirpath, _, _) in os.walk(os.path.expanduser("~/ray_results")):
+            if "checkpoint_{}".format(flags.checkpoint_freq) in dirpath \
+                and dirpath.split('/')[-3] == flags.exp_title:
+                checkpoint_path = os.path.dirname(dirpath)
+                checkpoint_number = -1
+                for name in os.listdir(checkpoint_path):
+                    if name.startswith('checkpoint'):
+                        cp = int(name.split('_')[1])
+                        checkpoint_number = max(checkpoint_number, cp)
+                break
```

Comment:
Probably remove this break (and indent what follows) to generate plots for all the grid-search values -- although that would upload a lot of curves on the leaderboard each time, I'm not sure if that's a problem

Reply:
It's not a problem as long as we name the results with the grid search values concatenated in some way

Reply:
It's just an extra half hour per grid point which is fine.

Reply:
Alright it's fixed